integration/js/utils/SdkBaseTest.js

```diff
@@ -90,19 +91,24 @@ class SdkBaseTest extends KiteBaseTest {
   }
 
   async createSeleniumSession(capabilities) {
-    const invalidSessionIdRegEx = new RegExp(/^new_request:/);
-    for (let i =0; i< 3; i++) {
-      const session = await SaucelabsSession.createSession(capabilities);
-      const sessionId = await session.getSessionId();
-      if (invalidSessionIdRegEx.test(sessionId)) {
-        console.log(`Invalid Saucelabs session id : ${sessionId}. Retrying: ${i+1}`);
-        await new Promise(r => setTimeout(r, 1000));
-      } else {
-        console.log(`Successfully create a Saucelabs session: ${sessionId}`);
-        return session;
+    if (process.env.SELENIUM_GRID_PROVIDER === "browserstack") {
+      const session = await BrowserStackSession.createSession(capabilities);
+      return session;
+    } else {
+      const invalidSessionIdRegEx = new RegExp(/^new_request:/);
+      for (let i =0; i< 3; i++) {
```

Comment:
nit: spaces

Reply:
should the value 3 be not based of the retry parameter?

Reply:
> nit: spaces

We should enable linting on the `/integration` folder.